serde-support/src/main/java/io/micronaut/serde/support/serdes/StringSerde.java

```diff
@@ -48,6 +48,11 @@ public String deserializeNullable(@NonNull Decoder decoder, @NonNull DecoderCont
         return decoder.decodeStringNullable();
     }
 
+    @Override
+    public boolean isAbsent(EncoderContext context, String value) {
+        return value == null;
```

Comment:
From the JsonInclude javadoc, I think this should be the default impl of isAbsent for all types: https://www.javadoc.io/doc/com.fasterxml.jackson.core/jackson-annotations/latest/com/fasterxml/jackson/annotation/JsonInclude.Include.html#NON_ABSENT

Reply:
Yeah it's the same as the super method